src/op_mode/show_configuration_json.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+#
+# Copyright (C) 2021 VyOS maintainers and contributors
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 or later as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+#!/usr/bin/env python3
+
+import argparse
+
+from vyos.configquery import ConfigTreeQuery
+from pprint import pprint
+
+
+config = ConfigTreeQuery()
+c = config.get_config_dict()
+
+parser = argparse.ArgumentParser()
+parser.add_argument("-p", "--pretty", action="store_true", help="Show pretty configuration in JSON format")
+
+
+if __name__ == '__main__':
+    args = parser.parse_args()
+
+    if args.pretty:
+        pprint(c)
```

Comment:
As @jestabro rightfully says, JSON export **must** always be done with `json.dumps` to produce syntactically correct JSON. Pretty-print from Python may look _superficially_ like JSON, but isn't.

Reply:
Also, `json.dump` to stdout is faster than `print(json.dumps(c))`.

Reply:
Sure, I removed pprint and use `indent=4` and  `json.dumps`
Thanks